### PR TITLE
Added export/import commands in Indy-Cli

### PR DESCRIPTION
**Pull request by Artemkaaas**
Signed-off-by: artem.ivanov <artem.ivanov@dsr-company.com>

### CODE REVIEW

File: cli/src/commands/wallet.rs [hunk 1]
```diff
@@ -247,15 +247,108 @@ pub mod delete_command {
     }
 }
 
+pub mod export_command {
+    use super::*;
+
+    command!(CommandMetadata::build("export", "Export opened wallet to the file")
+                .add_required_param("path", "Path to the export file")
+                .add_required_deferred_param("key", "Passphrase used to export key")
+                .add_example("wallet export path=/home/indy/export_wallet key=key")
```
Comment (vimmerru) [inline cli/src/commands/wallet.rs]: Rename key param to export_key

File: cli/src/commands/wallet.rs [hunk 2]
```diff
@@ -247,15 +247,108 @@ pub mod delete_command {
     }
 }
 
+pub mod export_command {
+    use super::*;
+
+    command!(CommandMetadata::build("export", "Export opened wallet to the file")
+                .add_required_param("path", "Path to the export file")
```
Comment (vimmerru) [inline cli/src/commands/wallet.rs]: Rename key param to export_path

File: cli/src/commands/wallet.rs [hunk 3]
```diff
@@ -247,15 +247,108 @@ pub mod delete_command {
...
+
+
+pub mod import_command {
+    use super::*;
+
+    command!(CommandMetadata::build("import", "Create new wallet and then import content from the specified file")
+                .add_main_param("name", "The name of new wallet")
+                .add_required_param("pool_name", "The name of associated Indy pool")
+                .add_required_deferred_param("key", "Auth key for the wallet")
+                .add_required_param("file", "Path to the file that contains exported wallet content")
```
Comment (vimmerru) [inline cli/src/commands/wallet.rs]: Rename file param to export_path

File: cli/src/commands/wallet.rs [hunk 4]
```diff
@@ -247,15 +247,108 @@ pub mod delete_command {
...
+                .add_required_deferred_param("import_key", "Passphrase used to export key")
```
Comment (vimmerru) [inline cli/src/commands/wallet.rs]: Rename import_key to export_key

File: cli/src/commands/wallet.rs [hunk 5]
```diff
@@ -247,15 +247,108 @@ pub mod delete_command {
...
+        trace!("execute >> ctx {:?} params {:?}", ctx, params);
+
+        let (wallet_handle, wallet_name) = ensure_opened_wallet(&ctx)?;
+
+        let export_path = get_str_param("export_path", params).map_err(error_err!())?;
+        let export_key = get_str_param("export_key", params).map_err(error_err!())?;
+
+        let export_config: String = json!({ "path": export_path.clone(), "key": export_key.clone() }).to_string();
+
+        trace!("Wallet::export_wallet try: wallet_name {}, export_path {}, export_key {}", wallet_name, export_path, export_key);
```
Comment (dkulic) [inline cli/src/commands/wallet.rs]: I am not sure if it is good idea to log export key?

Comment (vimmerru) [reply cli/src/commands/wallet.rs]: Yeah it is a bad idea. I believe we have this logging problem not only in this place.

**Pull request merged**